src/client/src/services/trades/__mocks__/creditTrades.ts

```diff
@@ -10,9 +10,9 @@ export const mockCreditTrades: CreditTrade[] = [
     counterParty: "J.P. Morgan",
     cusip: "10077LBC9",
     security: "ORCL 4.755 08/15/2026",
-    quantity: "10000",
+    quantity: 10000,
```

Comment:
this is ringing some bells about TODOs in the non-mock code about Hydra data ... ?